src/web/rustdoc.rs

```diff
@@ -367,6 +368,10 @@ pub fn rustdoc_html_server_handler(req: &mut Request) -> IronResult<Response> {
     // Get the latest version of the crate
     let latest_version = latest_release.version.to_string();
     let is_latest_version = latest_version == version;
+    let is_prerelease = semver::Version::parse(&version)
+        // should be impossible unless there is a bug in match_version
+        .map_err(|_| IronError::new(Nope::InternalServerError, status::InternalServerError))?
```

Comment:
Hmm, I think we could `impl From<Nope> for IronError` to make this easier ... probably should go in a separate PR though.

Reply:
Yep, I had the same thought :smile:

Reply:
Opened #1046.